css-device-adapt/Overview.bs

```diff
@@ -28,32 +28,29 @@ CSS 2.1 [[!CSS21]] specifies an
 <a href="https://www.w3.org/TR/CSS21/visudet.html#containing-block-details">
 initial containing block</a> for continuous media that has the dimensions
 of the <a href="https://www.w3.org/TR/CSS21/visuren.html#viewport">
-viewport</a>. Mobile/handheld device browsers have a viewport that is
-generally a lot narrower than a desktop browser window at a zoom level
-that gives a CSS pixel size
+viewport</a>. Since the viewport is generally no larger than the display,
+devices with smaller displays such as phones or tablets typically present
+a smaller viewport than larger devices like desktop or laptops.
+
+Unfortunately, many documents have historically been designed against larger
+viewports and exhibit a variety of bugs when viewed in smaller viewports.
+These include unintended layout wrapping, clipped content, awkward scrollable
+bounds, and script errors. To avoid these issues, mobile browsers generally
+use a fixed initial containing block width that mimics common desktop browser
+window size (typically 980-1024px). The resulting layout is then scaled down
+to fit in the available screen space.
+
+Although this approach mitigates the issues mentioned above, the downscaling
+means the CSS pixel size will be smaller than
 <a href="https://www.w3.org/TR/CSS21/syndata.html#length-units">recommended</a>
-by CSS 2.1.
-
-The narrow viewport is a problem for documents designed to look good in
-desktop browsers. The result is that mobile browser vendors use a fixed
-initial containing block size that is different from the viewport size,
-and close to that of a typical desktop browser window. In addition to
-scrolling or panning, zooming is often used to change between an overview
-of the document and zoom in on particular areas of the document to read
-and interact with.
-
-Certain DOCTYPEs (for instance XHTML Mobile Profile) are used to recognize
-mobile documents which are assumed to be designed for handheld devices,
-hence using the viewport size as the initial containing block size.
-
-Additionally, an HTML <code class=html>&lt;META&gt;</code> tag has been
-introduced for allowing an author to specify the size of the initial
-containing block, and the initial zoom factor directly. It was first
-implemented by Apple for the Safari/iPhone browser, but has since been
-implemented for the Opera, Android, and Fennec browsers. These
-implementations are not fully interoperable and this specification is
-an attempt at standardizing the functionality provided by the
-viewport <code class=html>&lt;META&gt;</code> tag in CSS.
+by CSS 2.1. Users will likely need to zoom on the content to view it
+comfortably.
+
+This mitigation is unneccessary for sites that have been designed to work
+well on small viewports.  This specification describes features which allow
+authors to control and opt-out of this behavior, including the HTML
+<code class=html>&lt;meta&gt;</code> viewport tag and CSS
```

Comment:
> Now, given that the meta viewport tag is certainly something that everybody needs to support, defining it normatively is certainly something we should look into doing. But so far we haven't, and I don't think the intro should be ahead of the spec.

Makes sense.  I'll update the example to use @viewport for now and edit the intro accordingly.  Opened #331 to cover the question of normative definition of the `<meta>` viewport tag.